lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -80,7 +80,7 @@ class ReportUIFeatures {
   /**
    * Adds export button, print, and other functionality to the report. The method
    * should be called whenever the report needs to be re-rendered.
-   * @param {LH.ReportResult} report
+   * @param {LH.Result} report
    */
   initFeatures(report) {
```

Comment:
should also drop the `Util.prepareReportResult` in `report-ui-features-test.js`

(that is a seriously anemic test file for the amount of jsdom setup it does :)